autoreduce_frontend/generate_token/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Are there actually going to be tests here or can this file be removed?

Reply:
Good spot, this can be removed, the selenium tests the functionality and the views are too basic for unit tests (they are just the django defaults)